multimodal/src/autogluon/multimodal/utils/hpo_trial.py

```diff
@@ -0,0 +1,20 @@
+import logging
```

Comment:
How about changing this file's name to `hpo.py`? All the hpo related util stuff can be put in this file.